runelite-client/src/main/java/net/runelite/client/plugins/achievementdiary/diaries/KandarinDiaryRequirement.java

```diff
@@ -96,6 +96,9 @@ public KandarinDiaryRequirement()
 			new SkillRequirement(Skill.MAGIC, 56));
 		add("Burn some Maple logs with a bow in Seers' Village.",
 			new SkillRequirement(Skill.FIREMAKING, 65));
+		add("Kill a Shadow Hound in the Shadow dungeon.",
+			new SkillRequirement(Skill.THIEVING, 53),
```

Comment:
I don't think quest reqs should be shown in the diary reqs like this.

Reply:
It makes sense in this case, since the quest only needs to be partially completed, and this skill is a part of the partial completion step of the quest.

Reply:
Then still you would have met the requirement for completing part of the quest, so my point still stands that if the req is met by doing the quest it shouldn't be a req on the diaries (unless its specific steal from x)

Reply:
As I understand it, the thieving requirement for the quest is not a requirement for completion, only for the specific part of the quest which is needed for this diary.